examples/cloudflare-workers/web/README.md

```diff
@@ -0,0 +1,53 @@
+# Using Momento inside a Cloudflare Worker
+
+This example uses
```

Comment:
We are going to want to update the title header on this page to clarify that this is different from the HTTP API example and probably link back to that one. We should also add some text here explaining what the difference is and what the tradeoffs are.